src/core/Form/SearchInput/SearchInput.test.tsx

```diff
@@ -5,7 +5,11 @@ import { axeTest } from '../../../utils/test/axe';
 import { SearchInput } from './SearchInput';
 
 const TestSearchInput = (
-  <SearchInput labelText="Test search input" data-testid="textinput" />
+  <SearchInput
+    labelText="Test search input"
+    data-testid="textinput"
+    id="test-id"
```

Comment:
As mentioned above, if the id is available like this and is used to access the component in DOM, what should the enduser expect to get, a label or an input? Or does it make any difference?